.changeset/bigquery.md

```diff
@@ -0,0 +1,3 @@
+'@finos/legend-studio': patch
```

Comment:
Could you use the script to generate this. This file is not of front matter format and will not be parsed properly

Reply:
Done